common/fileset-utils/src/main/resources/xml/xproc/fileset-load.xpl

```diff
@@ -292,7 +292,7 @@
                   </p:when>
 
                   <!-- XML -->
-                  <p:when test="$media-type='application/xml' or matches($media-type,'.*\+xml$')">
+                  <p:when test="matches($media-type,'.*/xml$') or matches($media-type,'.*\+xml$')">
```

Comment:
the expression could be slightly simplified with one `matches` using the `|` regex operator (but it's a matter of taste ;-)

Reply:
Missed that.

Reply:
Fixed